fastjsonschema/draft04.py

```diff
@@ -20,10 +20,7 @@ class CodeGeneratorDraft04(CodeGenerator):
     FORMAT_REGEXS = {
         'date-time': r'^\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d(?:\.\d+)?(?:[+-][0-2]\d:[0-5]\d|Z)?$',
         'email': r'^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$',
-        'hostname': (
-            r'^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*'
-            r'([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]{1,62}[A-Za-z0-9])$'
-        ),
+        'hostname': r'^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$',
```

Comment:
Yep, that regep is wrong, weird I haven't noticed never before. But maximum length is 63 and that should stay.

Reply:
Yes, you are correct. Somehow I forgot about that while looking through all the different suggestions I found online. I'll add that

Reply:
See https://github.com/horejsek/python-fastjsonschema/pull/27/commits/eb346dc11610bf57cf8826b41662de2d212f155c